### ISSUE TITLE
Workspace: Renaming result folder breaks eye icon

### ISSUE THREAD

**Issue by mshukla1** [PATRIC3/patric3_website#2180]
I ran a proteome comparison job. 
The job completed and the results look fine. 
Afterwords, I renamed the job result folder. 
Now, when I click on the EYE icon on the top right in the job result page, the circular viewer never loads. 
The debugger window shows workspace error. 

It looks like renaming job result folder breaks some of the links on the page, probably because it relies on the output name provided at the time of the job submission. 

<img width="1076" alt="screen shot 2019-01-22 at 8 57 37 am" src="https://user-images.githubusercontent.com/1236463/51545482-b45c0c80-1e27-11e9-8de0-39bcea638ea6.png">

**Comment by dawenx** [comment]
@mshukla1 how did you navigate to the page with "APS reference" in your screenshot? have you tried to clear the cache? When I navigated from PATRIC->PATRIC Workshop->Proteome comparison, I only saw the one with "APS ref". After clicking on that file, it displays correctly and the toggle icon (eye) is working for me. @nconrad can you try to navigate as well?

**Comment by nconrad** [comment]
yeah, maybe this was some sort of transient issue.  I copied that "- APS ref" file to my home, renamed it, and the view button still seems to work fine.

**Comment by dawenx** [comment]
should we close this issue then?

**Comment by dawenx** [comment]
plan to close this issue on this friday, let me know if you don't agree.

**Comment by mshukla1** [comment]
This is still an issue. 

Please run proteome comparison job, and then rename or move the job result, then click on the view / download icon at the top right, they don't work. 

I believe those links still trying to find the files with original path and name.
